### ISSUE TITLE
Improve Docker load/first time delay

### ISSUE THREAD

**Issue by rgaudin** [offspot/offspot-config#35]
First boot can be very long due to the delay it takes docker to ingest the images tar files.
We need to investigate ways to speed up this process.

- we are already using uncompressed tar for this
- there might be ways to replicate the output of what `docker load` does on the filesystem (nor sure what it does)
- we might be better of loading containers directly instead of images.

Note that the initial docker compose startup (when it creates the containers from the images) is fast.

**Comment by kelson42** [comment]
@rgaudin What about always booting only the very mandatory containee first (reverse proxy, captive portal, portal, ...) and then the rest. This wozld allow to have quicker visible tool all the time, not only at first start.

**Comment by rgaudin** [comment]
There is very little to gain here because starting existing containers is very fast and is done in parallel. That could be another ticket.

What really takes time is importing images from tar files into docker. We could use your approach to only import the non-content ones but that would still take a long time and then you reach the home page and click on links that raises a 502.

I think we'll need to optimize this import step at some point but we should also improve the fast-connect scenario. For instance at the moment, we start AP first then the docker stack. That's on purpose but it means someone connecting between AP-ready and docker-ready wont have the CP nor anything working during that time.

Let's keep this ticket focused on its subject (and maybe decide we don't want that optim)

**Comment by rgaudin** [comment]
Mentioning here that for Kiwix Hotspot H1 provisioned with https://github.com/offspot/provision-os/, the docker-load step happens at provisioning.
